packages/network/README.md

```diff
@@ -0,0 +1,3 @@
+# network
+
+This package contains networking-related classes and utilities.
```

Comment:
I'd like a more extensive Readme here following the format of other readmes for our packages. (Sorry I know this didn't even exist before so it's slightly better 😅)

- What is this package responsible for as a Cypress product, think - what would be completely broken if we deleted this package for example.

Also sections where relevant for `Installing`, `Building`, `Testing`, etc.

Reply:
Added in c90e72b11